scripts/linters/python_linter.py

```diff
@@ -45,30 +45,6 @@
 import pycodestyle # isort:skip  pylint: disable=wrong-import-order, wrong-import-position
 
 
-class StringMessageStream(python_utils.OBJECT):
```

Comment:
This custom stream wasn't working with pylint for some cases as it wasn't able to close the stream when needed (As this class doesn't handle closing and opening of the stream)